src/common/reducers/repositories-status.js

```diff
@@ -0,0 +1,34 @@
+import * as ActionTypes from '../actions/repository-input';
```

Comment:
In theory it could. At first it seemed easier to just change existing repository-input reducer to accept status of multiple repos.
Also it makes both reducers smaller and responding to less actions rather than one big one doing lot's of different things.

But that's exactly what I was thinking and what I'm about to mention as possible improvement  in an 'pre-holiday' email I'm about to send ;)